src/providers/bitcoin/unisatInjectedWallet.ts

```
@@ -0,0 +1,83 @@
+        const accountArray: string[] = await this.unisat.requestAccounts()
+        if (accountArray.length > 0) {
+          const address = accountArray[0]
+          this.address = address
+          resolve(true)
+        } else {
+          resolve(false)
+        }
+      } catch (err) {
+        reject(err)
+      }
+    })
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)